app/services/sign_in/login_redirect_url_generator.rb

```diff
@@ -2,25 +2,25 @@
 
 module SignIn
   class LoginRedirectUrlGenerator
-    attr_reader :client_id, :login_code, :client_state, :type
+    attr_reader :client_config, :login_code, :client_state, :type
 
     def initialize(user_code_map:)
       @login_code = user_code_map.login_code
       @type = user_code_map.type
-      @client_id = user_code_map.client_id
+      @client_config = user_code_map.client_config
       @client_state = user_code_map.client_state
     end
 
     def perform
-      redirect_uri = get_client_id_mapped_redirect_uri
+      redirect_uri = get_client_config_mapped_redirect_uri
       redirect_uri_params = get_redirect_uri_params
       redirect_uri.query = redirect_uri_params.to_query
       redirect_uri.to_s
     end
 
     private
 
-    def get_client_id_mapped_redirect_uri
+    def get_client_config_mapped_redirect_uri
```

Comment:
This is just a style opinion and a disabled [cop](https://github.com/rubocop/ruby-style-guide#accessor_mutator_method_names) in `vets-api` but it's to avoid using `get` on accessor and accessor-like methods (no parameters) because it's redundant. But I wonder if it would be cleaner to use an instance variable here? Thoughts?

```ruby
def perform
  redirect_uri.query = redirect_uri_params.to_query
  redirect_uri.to_s
end

private

def redirect_uri
  @redirect_uri ||= URI.parse(client_config.redirect_uri)
end

def redirect_uri_params
  ...
end
```